web/src/main/java/com/marklogic/hub/web/service/MappingManagerService.java

```diff
@@ -55,7 +58,7 @@
     @Autowired
     HubConfigImpl hubConfig;
 
-    private MappingValidator mappingValidator = null;
+    private Map<String, MappingValidator> mappingValidators = new HashMap<>();
```

Comment:
Does the Spring container get reloaded when a user logs out of QS to switch to another project? I'm wondering if this map would persist through that process, such that the wrong DatabaseClient objects would be used when testing mapping validations in the other project.

Reply:
The objects do persist in the map after a user logs out. I will add a method that will set the map to null in CurrentProjectController.onLogoutSuccess() method

Reply:
Updated the PR